lib/Twig/Lexer.php

```diff
@@ -136,7 +136,7 @@ public function tokenize($code, $filename = null)
             mb_internal_encoding($mbEncoding);
         }
 
-        return new Twig_TokenStream($this->tokens, $this->filename, $code);
+        return new Twig_TokenStream($this->tokens, $this->filename, $this->env->isDebug() ? $code : '');
```

Comment:
should it really be removed from the TokenStream itself ? IMO, this check belongs to the compilation step instead